dts/bindings/dma/dma.yaml

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2019, Song Qiang <songqiang1304521@gmail.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
```

Comment:
Could use a more compact header style (other files have been changed the same way):

```
# Copyright (c) 2019, Song Qiang <songqiang1304521@gmail.com>
# SPDX-License-Identifier: Apache-2.0

...
```